src/components/table.js

```diff
@@ -355,3 +399,295 @@ FlexTable.propTypes = {
   }),
   readOnly: PropTypes.bool
 }
+
+/**
+ * A virtual table with a fixed header and explicit column widths. Intended for displaying large
+ * datasets which may require horizontal scrolling.
+ */
+export const GridTable = forwardRefWithName('GridTable', ({
```

Comment:
`GridTable` is central to this PR. this object is copy/pasted from the DataTable class in terra-ui